wren-main/src/main/java/io/wren/main/connector/snowflake/SnowflakeClient.java

```diff
@@ -0,0 +1,119 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.wren.main.connector.snowflake;
+
+import com.google.common.collect.ImmutableList;
+import io.airlift.log.Logger;
+import io.wren.base.Column;
+import io.wren.base.Parameter;
+import io.wren.base.WrenException;
+import io.wren.base.config.SnowflakeConfig;
+import io.wren.connector.postgres.PostgresClient;
+import net.snowflake.client.jdbc.SnowflakeDriver;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.PreparedStatement;
+import java.sql.ResultSetMetaData;
+import java.sql.SQLException;
+import java.util.List;
+import java.util.Properties;
+
+import static io.wren.base.metadata.StandardErrorCode.GENERIC_USER_ERROR;
+import static io.wren.main.connector.snowflake.SnowflakeTypes.toPGType;
+import static java.util.Objects.requireNonNull;
+
+public class SnowflakeClient
+{
+    private static final Logger LOG = Logger.get(PostgresClient.class);
+
+    private final SnowflakeConfig config;
+
+    public SnowflakeClient(SnowflakeConfig config)
+    {
+        this.config = requireNonNull(config, "config is null");
+    }
+
+    public Connection createConnection()
+            throws SQLException
+    {
+        try {
+            Class.forName("net.snowflake.client.jdbc.SnowflakeDriver");
+            // TODO: Remove this line, it just use dependency to avoid dependency analyze failure
+            SnowflakeDriver driver = new SnowflakeDriver();
```

Comment:
It can be fixed by chaanging the `snowflake-jdbc` dependency scope to `runtime`.